fswatcher/fswatcher.go

```diff
@@ -157,15 +156,15 @@ func (fsw *fsWatcher) removeEventHandler(name string) error {
 
 func (fsw *fsWatcher) routeEvent(ev fsnotify.Event) {
 	evOp := ev.Op.String()
-	err := errors.New("")
+	var err error
 	switch evOp {
 	case "CREATE":
-		if fsw.isValidFile(ev.Name) {
+		if isValidFile(ev.Name) {
```

Comment:
if isValidFile return false, this will silently fail (it will do nothing). Maybe createEventHandler / writeEventHandler should check if it is valid, and if not return an error, instead of checking it here and doing nothing if it fails the check

Reply:
Maybe the function name  it's a bit misleading? `isValidFile` would indicate if it's a file we should add to the kv store. We are not storing folders or temporal/swap files. So I'm not sure if returning an error it makes sense here. wdyt?